CakePHP/Sniffs/PHP/SingleQuoteSniff.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace CakePHP\Sniffs\PHP;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Converts double quotes to single quotes for simple strings.
+ *
+ * @author Gregor Harlan <gharlan@web.de>
```

Comment:
These are awkward. Does he wanted us to email him about it?

Reply:
Why? No need to.

Reply:
r/whoosh

Reply:
?

Reply:
Thats normal MIT license copy.